### PR TITLE
added Philosophy (overview) section

### PR DESCRIPTION
**Pull request by cuttlefishh**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,3 +26,57 @@ source activate oecophylla
...
+
+#### IV. Assembly
+
+##### A. Contig assembly
+
+* [MetaSPAdes](http://bioinf.spbau.ru/en/metaspades)
+* [Megahit](http://www.metagenomics.wiki/tools/assembly/megahit)
+* [Quast](http://bioinf.spbau.ru/quast)
+
+##### B. Mapping and binning
```
Comment (qiyunzhu) [inline README.md]: mapping and binning are two different categories of tools

File: README.md [hunk 2]
```diff
@@ -28,3 +26,57 @@ source activate oecophylla
...
+
+* [Bowtie2](http://bowtie-bio.sourceforge.net/bowtie2/manual.shtml)
+* [Maxbin](https://sourceforge.net/projects/maxbin)
+
+##### C. Assembly analysis
```
Comment (qiyunzhu) [inline README.md]: Anvio should be binning analysis

File: README.md [hunk 3]
```diff
@@ -28,3 +26,57 @@ source activate oecophylla
...
+* [HUMAnN2](http://huttenhower.sph.harvard.edu/humann2)
+* [Shogun]()
+
+#### IV. Assembly
+
+##### A. Contig assembly
+
+* [MetaSPAdes](http://bioinf.spbau.ru/en/metaspades)
+* [Megahit](http://www.metagenomics.wiki/tools/assembly/megahit)
+* [Quast](http://bioinf.spbau.ru/quast)
```
Comment (qiyunzhu) [inline README.md]: Quast is assembly analysis.

File: README.md [hunk 4]
```diff
@@ -28,3 +26,57 @@ source activate oecophylla
...
+#### I. Quality control
+
+* [MultiQC](http://multiqc.info)
+* [Atropos]()
+
+#### II. Taxonomic analysis
+
+* [Kraken](https://ccb.jhu.edu/software/kraken/)
+* [MetaPhlAn2](http://huttenhower.sph.harvard.edu/metaphlan2)
+* [Shogun]()
```
Comment (qiyunzhu) [inline README.md]: Shogun: https://github.com/knights-lab/SHOGUN

**Review by qiyunzhu — COMMENTED** [review]
Great work! People feel more motivated see this! A few comments...

**Comment by tkosciolek** [comment]
Nice! Agree with @qiyunzhu comments.

**Comment by cuttlefishh** [comment]
Thanks @qiyunzhu! I will make those changes now.

File: README.md (hunk 2 above)
Comment (cuttlefishh) [reply README.md]: "Assembly" here is in the general sense, referring to the whole process. But point taken.

**Pull request merged**